app/MyApplicationLoader.scala

```diff
@@ -31,7 +31,7 @@ class MyApplicationLoader extends ApplicationLoader {
       new MyComponents(context).application
     } catch {
       case e: Throwable =>
-        SafeLogger.error(scrub"Failed to load application due to $e")
+        SafeLogger.error(scrub"Failed to load application due to", e)
```

Comment:
this didn't provide a stacktrace before, so when it failed , I couldn't tell it was failing while loading the play config.